converters/fromZigbee.js

```diff
@@ -5854,6 +5854,35 @@ const converters = {
             }
         },
     },
+    legrand_device_mode: {
+        cluster: 'manuSpecificLegrandDevices',
+        type: ['readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {};
+            const option0 = msg.data['0'];
+            // Beware that mode depends on device type
+            switch (option0) {
+            // contactor
+            case 0x0003:
```

Comment:
To  compact this please use an if else here:

e.g.
```js
if (option0 === 0x0003) payload.device_mode = 'switch';
else if (option0 === 0x0004) payload.device_mode = 'auto';
```

Reply:
I pushed a patch according your suggestions